ygnmi/ygnmi_test.go

```diff
@@ -408,26 +553,59 @@ func TestLookup(t *testing.T) {
 		}),
 	}}
 
-	for _, tt := range tests {
+	for _, tt := range nonLeafTests {
 		t.Run("nonleaf "+tt.desc, func(t *testing.T) {
 			tt.stub(fakeGNMI.Stub())
-			got, err := ygnmi.Lookup(context.Background(), c, tt.inQuery)
-			if diff := errdiff.Substring(err, tt.wantErr); diff != "" {
-				t.Fatalf("Lookup(ctx, c, %v) returned unexpected diff: %s", tt.inQuery, diff)
-			}
-			if err != nil {
-				return
-			}
-			verifySubscriptionPathsSent(t, fakeGNMI, tt.wantSubscriptionPath)
-			checkJustReceived(t, got.RecvTimestamp)
-			tt.wantVal.RecvTimestamp = got.RecvTimestamp
-
-			if diff := cmp.Diff(tt.wantVal, got, cmp.AllowUnexported(ygnmi.Value[*exampleoc.Parent_Child]{}), protocmp.Transform()); diff != "" {
-				t.Errorf("Lookup(ctx, c, %v) returned unexpected diff (-want,+got):\n %s\nComplianceErrors:\n%v", tt.inQuery, diff, got.ComplianceErrors)
-			}
+			lookupCheckFn(t, fakeGNMI, c, tt.inQuery, tt.wantErr, tt.wantSubscriptionPath, tt.wantVal)
 		})
 	}
 
+	t.Run("success ordered map", func(t *testing.T) {
```

Comment:
put this in the table?

Reply:
Can't do this because inQuery is
```inQuery              ygnmi.SingletonQuery[*exampleoc.Parent_Child]```
and the actual query is
```ygnmi.SingletonQuery[*exampleoc.Model_SingleKey_OrderedList_OrderedMap]```